### DISCUSSION TITLE
`public` inside `src/` not working in Vercel

### DISCUSSION THREAD

**Discussion by arvindell** [vercel/next.js#16674] [Help]
Hello. I am having problems when deploying to Vercel. My `public` directory is at `src/` (just like `pages/`), working well in local build. However, when I deploy to Vercel, I have to move my `public` directory to the root of my project or else the requests will fail with a 404.

Thanks in advance.

**Comment by shorn** [comment]
I think the framework doesn't want you to put the `/public` directory under `/src`.

I use `next export` and host my site elsewhere rather than using Vercel.
When I tried using `/src/public`, `next export` didn't pick up anything from the `/public` directory.  I had to move it back out to the root directory to get the export working properly.
